src/paho/mqtt/client.py

```diff
@@ -81,21 +81,10 @@
 else:
     EAGAIN = errno.EAGAIN
 
-# Python 2.7 does not have BlockingIOError.  Fall back to IOError
-try:
-    BlockingIOError
-except NameError:
-    BlockingIOError  = IOError
-
 MQTTv31 = 3
 MQTTv311 = 4
 MQTTv5 = 5
 
-if sys.version_info[0] >= 3:
-    # define some alias for python2 compatibility
-    unicode = str
```

Comment:
This cause some issue, we had reference to  `unicode` that need to be changed to `str`

The same happen for `basestring` that should be changed to `str` also.

Reply:
Done. Some tests still don't pass, but I think that's the case on `master` to.